obspy/taup/tests/test_plotting.py

```diff
@@ -24,39 +24,44 @@ def setUp(self):
         self.image_dir = os.path.join(os.path.dirname(__file__), 'images')
         self.model = TauPyModel(model="iasp91")
 
-    def test_spherical_many_phases(self):
+    def test_spherical_all_phases(self):
```

Comment:
TauP in this implementation has no concept of "all" phases - phase names are dynamically evaluated so "many" is IMHO not a bad name for the default set.